lib/utils/request.js

```diff
@@ -22,6 +23,34 @@ var os = require("os");
 var keys = require("../cmd/common/keys/user.js");
 var urlUtils = require('url');
 var once = require("once");
+var _ = require('underscore');
+
+
+/**
+ * Function To Generate A Relevant Error Message
+ * @param statusCode - HTTP Code For The Error Message
+ * @param body - Error Response Body
+ */
+function createErrorMessage(statusCode, body){
+
+  var msg = 'Error - not 2xx status code. (' + statusCode + ')\n';
+
+  //If the body is a string, try to parse it as an object.
+  if(_.isString(body)){
+    try {
+      body = JSON.parse(body);
+    } catch(e){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AEmptyBlock)